librz/debug/pid.c

```diff
@@ -108,7 +108,7 @@ RZ_API int rz_debug_thread_list(RzDebug *dbg, int pid, char fmt) {
 				pj_o(j);
 				pj_kb(j, "current", dbg->tid == p->pid);
 				pj_ki(j, "pid", p->pid);
-				pj_ks(j, "status", &p->status);
+				pj_ks(j, "status", strdup(&p->status));
```

Comment:
doesn't pj_ks copy `p->status` ?

Reply:
This is what Coverity says:

![image](https://user-images.githubusercontent.com/203261/114397025-d12fb680-9bd0-11eb-8482-89a78abc08b0.png)

Reply:
i think is a false positive
because then it goes to snprintf

Reply:
```c
RZ_API PJ *pj_se(PJ *j, const char *k) {
	rz_return_val_if_fail(j && k, j);
	pj_comma(j);
	if (j->str_encoding == PJ_ENCODING_STR_ARRAY) {
		pj_raw(j, "[");
	} else {
		pj_raw(j, "\"");
	}
	char *en = rz_str_encoded_json(k, -1, j->str_encoding);
	if (en) {
		pj_raw(j, en);
		free(en);
	}
	if (j->str_encoding == PJ_ENCODING_STR_ARRAY) {
		pj_raw(j, "]");
	} else {
		pj_raw(j, "\"");
	}
	return j;
}
```

Reply:
Removed that commit completely.